samples/name_domains_values/Coded_Domain_Land_Unit_SpecialEntities.csv

```diff
@@ -0,0 +1,5 @@
+Name,Value
```

Comment:
Note: the current domain name in  is "Land_Unit_Special_Entity_Subtype" (also the name used for the issue) so you may have to adjust accordingly if you use this file name (since this is where the domain name is derived from).